app/src/main/java/org/wikipedia/dataclient/page/PageSummary.java

```diff
@@ -32,6 +40,30 @@
     @Nullable @SerializedName("originalimage") private Thumbnail originalImage;
     @Nullable private String lang;
     private int pageid;
+    @SuppressWarnings("unused,NullableProblems") @NonNull private String displaytitle;
+    @SuppressWarnings("unused") @Nullable private String redirected;
```

Comment:
Does the `redirected` field exist in the Summary response?

Reply:
No @dbrant Summary response doesn't have redirected. However, we can get rid of the logic of checking for that string since we uniformly create the PageTitle object with the canonical title.